vowpalwabbit/cs_active.cc

```diff
@@ -336,18 +335,17 @@ base_learner* cs_active_setup(VW::setup_base_i& stack_builder)
   data->t = 1;
 
   auto loss_function_type = all.loss->getType();
-  if (loss_function_type != "squared") THROW("error: you can't use non-squared loss with cs_active");
+  if (loss_function_type != "squared") THROW("You can't use non-squared loss with cs_active");
```

Comment:
In the non-aggressive effort should we avoid `You` altogether? i.e. `Non-squared loss can not be used with cs_active`?

Reply:
Definitely, thanks for catching this